apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -271,7 +271,7 @@ function load_block_patterns() {
 
 	Block_Patterns::get_instance();
 }
-add_action( 'plugins_loaded', __NAMESPACE__ . '\load_block_patterns' );
+add_action( 'init', __NAMESPACE__ . '\load_block_patterns', 20 );
```

Comment:
Does this just need to load before the Gutenberg plugin? If so you can do something similar like we did for site editor: https://github.com/Automattic/wp-calypso/pull/45499/files#diff-1d0b2aecea674ad2a822870edfb5b4a1L54

If not, why is the priority 20 here? A comment explaining it could be helpful.

Reply:
It should run after core registers the core patterns, so that they can be removed.